### DISCUSSION TITLE
How to reactively validate two or more fields on the stateUpdated of another

### DISCUSSION THREAD

**Discussion by klivinout** [filamentphp/filament#10279] [Help]
i have two fields Organisme : Select and Exercice : TextInput and i validate each one reactively if the duo(organisme_id, exercice) exists in database: 
![image](https://github.com/filamentphp/filament/assets/11075412/082d7adc-efe5-4346-974f-2f883768d846)

And : 
![image](https://github.com/filamentphp/filament/assets/11075412/c764ef56-bab8-421e-87a7-a9290749201f)
it works only partially, because the changing of the value of Organisme doesn't trigger the revalidation of Exercice, how can i sort that? thank you.

**Comment by leandrocfe** [comment]
Maybe checking all fields using `$livewire->form->getState()` ?
